run/GCHP/setCommonRunSettings.sh.template

```diff
@@ -234,41 +249,125 @@ if [[ "x${dustDead}" == "xon" || "x${tomasDustDead}" == "xon" ]]; then
 	echo "${msg}"
     fi
 
-    if [[ $CS_RES -eq 24 ]]; then
-	if [[ "x${metField}" == "xMERRA2" ]]; then
-	    Dust_SF=5.4856e-5
-	elif [[ "x${metField}" == "xGEOSFP" ]]; then
-	    Dust_SF=4.6495e-5
-	elif [[ "x${metField}" == "xGEOSIT" ]]; then
-	    Dust_SF=3.7410e-5
-	else
-	    msg="Dust scale factors are currenlty only available for C24. "
-	    msg+="with MERRA-2, GEOSFP, or GEOSIT."
-	    echo "${msg}"
-	    exit 1
-	fi
-    #
-    # NOTE: Met-field dependent mass tuning factors for the grid resolutions
-    # listed below will be added later.  Leave the prior values for now.
-    #
-    elif [[ $CS_RES -eq 30 ]]; then
-	Dust_SF=5.76e-4 # approximated as linear interpolation between 24 and 48. use for testing only!
-    elif [[ $CS_RES -eq 48 ]]; then
-        Dust_SF=5.0416e-4
-    elif [[ $CS_RES -eq 90 ]]; then
-        Dust_SF=4.0e-4
-    elif [[ $CS_RES -eq 180 ]]; then
-        Dust_SF=3.23e-4
-    elif [[ $CS_RES -eq 360 ]]; then
-        Dust_SF=2.35e-4
-    elif [[ $CS_RES -eq 720 ]]; then
-        Dust_SF=2.3e-4
-    else
-	msg="Dust scale factor not defined for this resolution. Please add "
-	msg+="the tuning factor you wish to use for the target resolution "
-	msg+="above."
-	echo "${msg}"
-        exit 1
+    #------------------------------------------------------------------------
+    # Dust mass tuning factors for DustL23M were generated by @1Dandan, see:
```

Comment:
This is the first time I've seen github handle in the code. My preference is name but we can discuss.

Reply:
@lizziel: I've added the name in commit a4fdcdcba.